src/components/PlanetInfo/PlanetInfo.tsx

```diff
@@ -86,24 +127,75 @@ const PlanetInfo: React.FC<InfoProps> = ({ currentPlanet, resetSort }): JSX.Elem
+                {(name === 'Mars' || name === 'Jupiter' || name === 'Saturn' || name === 'Uranus' || name === 'Neptune') &&
+                  <>
+                    <span className='distance-multiplier'>{ (distance_from_sun / 149598262).toFixed(2)}</span> 
+                    <span className='distance-words'> times smaller </span>
+                  </>
+                }
+                {' '} from {name} 
+              </p>
+            }
           </div>
           <div className='planet-info-card planet-info-card__moons'>
             <img className='info-icon info-icon__moons' alt='eclipse icon' src='../space/eclipse-fill.svg'></img>
-            <p className='planet-info-text planet-info-text__moons'>On {name} you {name !== 'Earth' && 'would'} see <span className='moon-count'>{number_of_moons}</span> <span className='moon-word'>{getMoonWord(number_of_moons)}</span> in the sky</p>
+            <p className='planet-info-text planet-info-text__moons'>
+              On {name} you {name !== 'Earth' && 'would'} see {' '}
+              <span className='moon-count'>{number_of_moons}</span> 
+              <span className='moon-word'> {getMoonWord(number_of_moons)}</span> 
+              {' '} in the sky
+            </p>
           </div>
           <div className='planet-info-card planet-info-card__day'>
-            <img className='info-icon info-icon__length-of-day' alt='spinning planet icon' src='../space/orbit-fill.svg'></img>
-            <h2 className='planet-info-title planet-info-title__length-of-day'>Length of day</h2>
-            <h3 className='planet-info-length-of-day'>{length_of_day} hours</h3>
+            <img className='info-icon info-icon__length-of-day' alt='spinning planet icon' src='../space/planet-fill.svg'></img>
+            {name === 'Earth' && 
+              <>
+                <h2 className='planet-info-title planet-info-title__length-of-day'>Length of day</h2>
+                <p className='planet-info-text planet-info-text__length-of-day'>
+                  <span className='length-of-day-earth'>{length_of_day} hours</span>
+                </p>
+              </>
+            }
+            {name !== 'Earth' && 
+              <p className='planet-info-text planet-info-text__length-of-day'>
+                Compared to kids on Earth, kids on {name} would have {' '}
+                {(name === 'Mars' || name === 'Mercury' || name === 'Venus') && 
+                  <>
+                    <span className='length-of-day-multiplier'>{(length_of_day - 23.9).toFixed(1)}</span> 
+                    <span className='length-of-day-words'> extra hours </span> 
+                  </>
+                }
+                {(name === 'Jupiter' || name === 'Saturn' || name === 'Neptune' || name === 'Uranus') && 
+                  <>
+                    <span className='length-of-day-multiplier'>{(23.9 - length_of_day).toFixed(1)}</span> 
+                    <span className='length-of-day-words'> fewer hours </span> 
```

Comment:
Way to go with all of this conditional rendering, so many different conditions to check --- you crushed it!